### PR TITLE
Rename `queueLength` to `messageCount` for Azure Service Bus scaler

### LINKED ISSUE #1109: Rename `queueLength` to `messageCount` for Azure Service Bus
Rename `queueLength` to `messageCount` or `depth` for Azure Service Bus given it's a confusing name if you are using Service Bus topics.

Alternative approach would be to introduce `subscriptionLength` but that feels odd and requires you to change.

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Rename `queueLength` to `messageCount` for Azure Service Bus scaler since this is confusing when using topics.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [x] Changelog has been updated

I'll open a doc PR when this is agreed and merged. ([depends on this PR as well](https://github.com/kedacore/keda-docs/pull/251))

Relates to #1109

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Didn't run them locally and don't know how to write Go, but this will work for sure!

File: pkg/scalers/azure_servicebus_scaler.go [hunk 1]
```diff
@@ -3,9 +3,10 @@ package scalers
 import (
 	"context"
 	"fmt"
-	"github.com/kedacore/keda/pkg/scalers/azure"
 	"strconv"
 
+	"github.com/kedacore/keda/pkg/scalers/azure"
+
```
Comment (zroubalik) [inline pkg/scalers/azure_servicebus_scaler.go:9]: please remove the empty line 9

File: pkg/scalers/azure_servicebus_scaler.go [hunk 2]
```diff
@@ -3,9 +3,10 @@ package scalers
...
 	servicebus "github.com/Azure/azure-service-bus-go"
```
Comment (zroubalik) [inline pkg/scalers/azure_servicebus_scaler.go:11]: Please remove the empty line 11

Comment (zroubalik) [inline pkg/scalers/azure_servicebus_scaler.go:11]: Please remove the empty line 11

**Review by ahmelsayed — APPROVED** [review]
LGTM!!

**Comment by ahmelsayed** [comment]
actually looks like a test case (`TestAzServiceBusGetMetricSpecForScaling`) is failing. I think the order of the test data in `parseServiceBusMetadataDataset` has changed, but `azServiceBusMetricIdentifiers`, which is used in that test depends on that order.

**Pull request merged**